marketpulse/settings/base.py

```diff
@@ -171,5 +171,6 @@
 MOZILLIANS_API_KEY = config('MOZILLIANS_API_KEY', default=None)
 
 # Mapbox API
+MAPBOX_GEOCODE_URL = 'http://api.tiles.mapbox.com/v4/geocode/mapbox.places/'
```

Comment:
It doesn't really make sense for me since our code works with this specific endpoint. Any other version will fail.